online-fishing-store/src/app/pages/profile-page/personal-data/personal-data.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { InputGenderLabel } from 'src/app/enums/input-gender-label-enum';
+import { UserData } from 'src/app/interfaces/user-data';
+import { UserService } from 'src/app/services/users/user.service';
+import { COUNTRYES } from './countryes';
+
+@Component({
+  selector: 'app-personal-data',
+  templateUrl: './personal-data.component.html',
+  styleUrls: ['./personal-data.component.scss']
+})
+export class PersonalDataComponent implements OnInit {
+  public avatarImgAltAttributeValue: string = 'photo';
+  public profilePicture: string = 'assets/img/profile_picture.png';
+  public readonly inputGenderLabelMale: string = InputGenderLabel.MALE;
+  public readonly inputGenderLabelFemale: string = InputGenderLabel.FEMALE;
+  public countryes: string[] = COUNTRYES;
+  public form: FormGroup;
+  public userData: UserData;
+  public isEditAvatar: boolean = true;
+  public isEditFirstName: boolean = true;
+  public isEditLastName: boolean = true;
+  public isEditCountry: boolean = true;
+  public isEditAddress: boolean = true;
+  public isEditPhoneNumber: boolean = true;
+  public isEditGender: boolean = true;
+  public isEditedForm: boolean = true;
+
+  constructor(
+    public userService: UserService
+  ) { }
+
+  public getUser(): void{
+    this.userService.getUser().subscribe((user: UserData) => {
+        this.userData = user;
+      })
+  }
+
+  ngOnInit(): void {
+    this.getUser()
+
+    this.form = new FormGroup({
+      avatar: new FormControl(''),
```

Comment:
Why you set empty value if you have user with filled properties like firstName, lastName end etc..

You should work with FormGroup instead of [(ngModel)]